db_test.go

```diff
@@ -757,6 +757,43 @@ func TestSort(t *testing.T) {
 	})
 }
 
+func TestImportExportCollection(t *testing.T) {
```

Comment:
Test must be more robust. I want this flow to be implemented:

- In the runCloverTest function, specify the todosPath, so that the todos collection is loaded automatically.
- Run `ExportCollection` to export the loaded `todos` collection into a json file.
- Run the  `ImportCollection` function and import the exported json file under a different collectionName (`todosImport` is fine).
- Check that the two collection `todos` and `importTodos` are equal by comparing corresponding documents